File: src/main/java/boxes/BoxTester.java
package boxes;

import javax.swing.*;
import java.util.Scanner;

public class BoxTester {
    public static void main(String[] args) {
        String letter;
        float length;
        float width;
        int height;
        Scanner sca=new Scanner(System.in);
        Box3 box3 =new Box3();
        Box5 box5 =new Box5();
        System.out.println("plz choice Box: (A) Box3 / (B) Box5");
        letter= sca.next();
        if(letter.equals("A")){
            System.out.println("plz enter object's length:");
            length= sca.nextFloat();
            System.out.println("plz enter object's width:");
            width=sca.nextFloat();
            System.out.println("plz enter object's height");
            height=sca.nextInt();

            if(box3.validate(length,width,height)==true){
                System.out.println("Box3 OK!!");
            }
            else {
                System.out.println("Your object too big!!");
                }
            }
        else if (letter.equals("B")){
            System.out.println("plz enter object's length:");
            length= sca.nextFloat();
            System.out.println("plz enter object's width:");
            width=sca.nextFloat();
            System.out.println("plz enter object's height");
            height=sca.nextInt();
            if(box5.validate(length,width,height)==true){
                System.out.println("Box5 OK!!");
            }
            else {
                System.out.println("Your object too big!!");
            }
        }
        else{
            System.out.println("plz choice the box!!");
        }

    }
}

File: src/main/kotlin/boxes/BoxTester.kt
package com.boxes

import sun.security.util.Length

fun main(args:Array<String>) {
    val letter :String
    var length : Float
    var width : Float
    var height : Int
    var box3=Box3()
    var box5=Box5()
    println("plz choice Box : (A) Box3/(B) Box5")
    letter= readLine().toString()
    if (letter == "A"){
        println("plz enter object's length:")
        length= readLine()!!.toFloat()
        println("plz enter object's width:")
        width= readLine()!!.toFloat()
        println("plz enter object's height")
        height= readLine()!!.toInt()
        if (box3.validate(length,width,height)==true){
            println("Box3 OK!!")
        }
        else{
            println("Your object too big!!")
        }
    }
    else if(letter == "B"){
        println("plz enter object's length:")
        length= readLine()!!.toFloat()
        println("plz enter object's width:")
        width= readLine()!!.toFloat()
        println("plz enter object's height")
        height= readLine()!!.toInt()
        if (box5.validate(length,width,height)==true){
            println("Box5 OK!!")
        }
        else{
            println("Your object too big!!")
        }
    }
    else{
        println("plz enter A/B!!")
    }
}



class Box3(){
    var length : Float = 0.0f
    var width : Float = 0.0f
    var height : Int = 0
    fun validate(length: Float,width:Float,height:Int) : Boolean{
        return !(length>23||width>14||height>13)
    }

}

class Box5(){
    var length : Float = 0.0f
    var width : Float = 0.0f
    var height : Int = 0
    fun validate(length: Float,width:Float,height:Int) : Boolean{
        return !(length>39.5||width>27.5||height>23)
    }

}